var/spack/repos/builtin/packages/blis/package.py

```diff
@@ -9,38 +9,19 @@
 # https://github.com/flame/blis/issues/197
 
 
-class BlisBase(Package):
+class BlisBase(MakefilePackage):
```

Comment:
FWIW. I was going to ask why you didn't switch to `AutotoolsPackage` so you could take advantage of some of the helper methods below until I read https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#important-files and looked at the `configure` script in the software repository.

Reply:
Yep, it seems a lot of people like to have custom scripts called `configure` :laughing: